tests/unit/client/feedback/dataset/test_local.py

```diff
@@ -94,3 +94,39 @@ def test_add_records_validation_error(record: FeedbackRecord, exception_cls: Exc
     with pytest.raises(exception_cls, match=exception_msg):
         dataset.add_records(record)
     assert len(dataset.records) == 0
+
+
+def test_sort_by_for_local_is_not_implemented():
```

Comment:
can we create good first issue work items to support this?

Reply:
ditto